xcode/Subconscious/Shared/Components/BacklinksView.swift

```diff
@@ -30,7 +30,7 @@ struct BacklinksView: View {
                             .buttonStyle(RelatedNoteButtonStyle(color: entry.color))
                         }
                         .tint(
-                            entry.headers.themeColor?.toHighlightColor()
+                            entry.highlightColor
```

Comment:
I'm fuzzy on the control flow here, but it seems like we're bypassing the header-set color?

Reply:
These computed properties check the headers first before falling back to deriving the color from the address, so the old behaviour is preserved (and extended):

```swift
extension EntryStub {
    var color: Color {
        self.headers.themeColor?.toColor()
            ?? self.address.themeColor.toColor()
    }
    
    var highlightColor: Color {
        self.headers.themeColor?.toHighlightColor()
            ?? self.address.themeColor.toHighlightColor()
    }
}
```

Reply:
Nice!